### ISSUE TITLE
Let user download a zipped folder containing everything she needs for a tutorial in one hit

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/DataScienceTutorials.jl#173]
It's kind of tedious to copy and past all those files. Can't we download a zipped folder with this structure:

```
Manifest.toml
Project.toml
tutorial.jl (annotated)
tutorial.ipynb
```

**Comment by tlienart** [comment]
Yes good idea, this should be doable within the GitHub action itself: after the build, it could go through the `__site` folder, form zips and put them in the same location, I'll give it a shot if I get some time this week

**Comment by ablaom** [comment]
Awesome!
